dracut.sh

```diff
@@ -1295,6 +1296,13 @@ else
     exit 1
 fi
 
+if [[ $persistent_policy == "mapper" ]]; then
+    unset persistent_policy
```

Comment:
Perhaps you can also consider not to unset the variable. I would find that easier to read.

Reply:
IMHO, this is the least intrusive change, since the effect of the new "mapper" option is to not set any value for the `persistent_policy` variable.